Changes

```diff
@@ -26,6 +26,11 @@ Working version
 
 ### Other libraries:
 
+- #10084: Unix.open_process_args_* functions now look up the program in the PATH
```

Comment:
The glob only matches 3 of the 4 functions because of the trailing underscore 😈

Reply:
This is a breaking change, strictly speaking, so it should be listed with a `*` instead of a `-`.

Why is this breaking?  Assume the current directory contains an executable named `foo.exe`, and `PATH` does not contain `.`, like it should.  `Unix.open_process_args "foo.exe" ...` succeeds with the current implementation and fails with the new implementation.

Reply:
Good point, I amended the `Changes` entry and the docstring for the function.